USE_CASES.md

```diff
@@ -169,3 +169,17 @@ print(response.status_code)
 print(response.body)
 print(response.headers)
 ```
+
```

Comment:
Please add the `Table of Contents` entries and change `sendgrid-csharp` in the URLs to `sendgrid-python`.

Thanks!

Reply:
fixed